stdlib/experimental/query/from.flux

```diff
@@ -0,0 +1,46 @@
+package query
+
+fromRange = (bucket, start, stop=now()) =>
+    from(bucket: bucket)
+        |> range(start: start, stop: stop)
+
+filterMeasurement = (table=<-, measurement) => table |> filter(fn: (r) => r._measurement == measurement)
+
+filterFields = (table=<-, fields=[]) =>
+    if length(arr: fields) == 0 then
+        table
+    else
+        table |> filter(fn: (r) => contains(value: r._field, set: fields))
+
+inBucket = (
+    bucket,
+    measurement,
+    start,
+    stop=now(),
+    fields=[],
+    predicate=(r) => true
+) =>
+    fromRange(bucket: bucket, start: start, stop: stop)
+        |> filterMeasurement(measurement)
+        |> filter(fn: predicate)
+        |> filterFields(fields)
+
+
+// counter funcs
+_rateDet = (table=<-, unit=0s, nonNegative=true) =>
+    if string(v: unit) == "0s" then
+        table |> difference(nonNegative: nonNegative, columns: ["_value"])
+    else
+        table |> derivative(unit: unit, nonNegative: nonNegative, columns: ["_value"], timeColumn: "_time")
+
+_sumWindowDet = (table=<-, window=0s, fn=mean) =>
+    if string(v: window) == "0s" then
+        table
+    else
+        table |> aggregateWindow(every: window, fn: fn)
+
+aggregateCounter = (table=<-, pollingInterval=0s, window, groupCols=[]) =>
+    table
+        |> _rateDet(unit: pollingInterval, nonNegative: true)
+        |> group(columns: groupCols)
+        |> _sumWindowDet(window: window)
```

Comment:
I don't think this does what we want yet. Can you add a test case that shows how the `aggregateCounter` function is used? That will help me understand the intent of the function.

To add a test case see one of the other files that end int `_test.flux`. The necessary parts are a `test` key word statement which needs input data and expected output data for a given function. 

The part that doesn't look correct yet is the `group |> aggregeateWindow` part. I would have expected `aggergateWindow` to not work for this case because there are two separate aggregation steps. I could just be reading the intent wrong, which is where a test case comes in.